raft/raft.go

```diff
@@ -596,6 +608,11 @@ func stepLeader(r *raft, m pb.Message) {
 			// drop any new proposals.
 			return
 		}
+		if r.leadTransferee != None {
```

Comment:
I believe @bdarnell wont want to see this to happen in cockroach... at least, we should add a todo to buffer these proposals?